<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetriProperties</class>
 <widget class="QWidget" name="PetriProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>288</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>7</number>
     </property>
     <widget class="QWidget" name="place">
      <layout class="QGridLayout" name="gridLayout_17">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Lugar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>149</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_place_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Fichas:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_place_tokens"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>530</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fplace">
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Lugar fluido</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>138</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="le_fplace_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Fichas:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="le_fplace_tokens"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Valor limite:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="le_fplace_limit"/>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Valor de quebra:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="le_fplace_break"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>189</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="itrans">
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Transição imediata</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>123</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="le_itrans_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Prioridade:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="le_itrans_prior"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Guarda:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="le_itrans_guard"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>Probabilidade:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QComboBox" name="cb_itrans_prob">
            <item>
             <property name="text">
              <string>Constante</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Lugar dependente</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Função</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Valor:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_itrans_prob_value"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Lugar:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cb_itrans_prob_place">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ttrans">
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Transição temporizada</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" rowspan="2" colspan="2">
          <widget class="QLineEdit" name="le_ttrans_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Guarda:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="le_ttrans_guard"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Prioridade:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="le_ttrans_prior"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Definição da distribuição:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Distribuição:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cb_ttrans_distr">
                <item>
                 <property name="text">
                  <string>Exponential</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Constant</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Uniform</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>TruncatedNormal</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Weibull</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>LogNormal</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Geometric</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Erlang</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Normal</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>HyperExponencial</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QPlainTextEdit" name="pte_ttrans_distr"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Taxa:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="le_ttrans_rate"/>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QComboBox" name="cb_ttrans_rate">
              <item>
               <property name="text">
                <string>Constante</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lugar dependente</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Função</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>Lugar:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cb_ttrans_place">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Política:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cb_ttrans_pol"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Afetado:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cb_ttrans_afected"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>35</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="arc">
      <layout class="QGridLayout" name="gridLayout_19">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Arco</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>221</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>De:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_arc_from">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Para:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_arc_to">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="0" column="0" colspan="2">
          <widget class="QRadioButton" name="rb_arc_const">
           <property name="text">
            <string>Constante</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="rb_arc_function">
           <property name="text">
            <string>Função</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Multiplicidade:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="le_arc_mult"/>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>388</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="farc">
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Arco fluido</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="inhibitor">
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>59</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Inibidor</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="net">
      <layout class="QGridLayout" name="gridLayout_20">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Rede</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>189</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_net_name"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>439</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
